composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "zendframework/zend-http": "~2.2.5",
-        "zendframework/zend-stdlib": "~2.2.5",
-        "zendframework/zend-uri": "~2.2.5"
+        "php": ">=5.5.0",
+        "zendframework/zend-stdlib": "~2.2.5"
     },
     "require-dev": {
         "phpunit/PHPUnit": "3.7.*",
         "satooshi/php-coveralls": ">=0.6.0",
+        "psr/http-message": "1.0.*@dev",
```

Comment:
@bakura10 according to me, you should never install `zend/http` or `guzzle/http `. Let the users choose what they needs, but install `psr7/http-message` ie it :) . So if I have guzzle already installed I can make use of Dash with guzzle or any other implementations.